app/views/submissions/steps/trial/_step03.html.haml

```diff
@@ -31,52 +26,56 @@
     = content.radio_button :lines_or_varieties, 'plant_varieties'
     Plant varieties
   %small.help-block
-    Per each plant scoring unit submitted, the source of genetic material should be provided. It may either be a
-    %b plant line
-    or a
-    %b plant variety (cultivar).
-    This choice depends on the type of experiment you are conducting. You should select the
-    %b Plant lines
-    option if you
+    Per each plant scoring unit submitted, the source of genetic material should be provided.
+    It may either be a <b>plant line</b> or a <b>plant variety (cultivar)</b>.
+    This choice depends on the type of experiment you are conducting. You should select
+    the <b>Plant lines</b> option if you
     %ul
       %li created new genetic material (using, for instance, crossing or mutagenesis) in the course of your project,
       %li or you use former plant line genetic material which is not (yet) recognized as a variety or a cultivar.
-    In other cases you should probably select the the
-    %b Plant varieties
-    option.
+    In other cases you should probably select the the <b>Plant varieties</b> option.
 
 - if submission.raw_data?
   %fieldset
     %legend Technical replicate numbers
     .alert.alert-info
       %p
-        If you are using
-        %i technical replicates
-        in your experiment, please set the number of replicates you are scoring per trait. Leaving
-        the value at
-        %b 1
-        means you are not using technical replicates for a given trait.
+        If you are using <i>technical replicates</i> in your experiment, please
+        set the number of replicates you are scoring per trait. Leaving the value
+        at <b>1</b> means you are not using technical replicates for a given trait.
 
       - if submission.sorted_trait_names.size > 1
         %p
           You can have varying number of technical replicates for different traits.
 
-    - submission.sorted_trait_names.each_with_index do |trait_name, idx|
+    - sorted_trait_names = submission.sorted_trait_names
+    - col_count = [4, sorted_trait_names.size].min
+    - col_size = 12 / col_count
+    - row_count = (sorted_trait_names.size / col_count.to_f).ceil
+
+    - (0...row_count).each do |row_idx|
```

Comment:
Perhaps `sorted_trait_names.each_slice(col_count) do |row_traits|` would work here? Then, in the inner "row" context - a loop over `row_traits`. You might not need the abominable arithmetic operator this way ;).